dotCMS/src/integration-test/java/com/dotcms/util/TestDataSourceProvider.java

```diff
@@ -36,6 +37,14 @@ private Properties getProperties()  {
 
         try (InputStream resourceStream = loader.getResourceAsStream(dbType + "-db-config.properties")) {
             properties.load(resourceStream);
+            
+            Logger.info(this.getClass(), "Found TEST DB properties:");
+            for(Map.Entry<Object,Object> prop : properties.entrySet()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'prop' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)